com.unity.multiplayer.mlapi/Tests/Editor/NetworkBehaviourTests.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using NUnit.Framework;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace MLAPI.EditorTests
+{
+    public class NetworkBehaviourTests
+    {
+        [Test]
+        public void HasNetworkObjectTest()
+        {
+            var gameObject = new GameObject(nameof(NetworkObject));
```

Comment:
```suggestion
            var gameObject = new GameObject(nameof(HasNetworkObjectTest));
```

minor: I'd like to suggest the `GameObject` as `HasNetworkObjectTest` instead of super-generic `NetworkObject`